src/foam/core/types.js

```diff
@@ -115,6 +115,15 @@ foam.CLASS({
   label: 'Date and time'
 });
 
+foam.CLASS({
+  package: 'foam.core',
+  name: 'Time',
+  extends: 'String',
```

Comment:
Why does it extend String instead of extending Date?

Reply:
A time is not a date. A date is a moment in time but a time is a moment in a day. There are an infinite number of dates with the same time.